pip/cmdoptions.py

```diff
@@ -498,6 +498,18 @@ def only_binary():
          "options to setup.py install. If you are using an option with a "
          "directory path, be sure to use absolute path.")
 
+download_options = partial(
```

Comment:
Line 49 of `download.py`: `cmd_opts.add_option(cmdoptions.download_options())`
I copied this paradigm from `install.py` because of how similar `pip download` is to `pip install`. If you think this should not be included I will remove it.